example/line-plot-custom-color.md

```diff
@@ -0,0 +1,48 @@
+# Line Plot Custom Color
+
+You can set custom colors for each line in a [LinePlot](http://d3plus.org/docs/#LinePlot) by setting the `Line.stroke` property inside [.shapeConfig( )](http://d3plus.org/docs/#Plot.shapeConfig):
+
+```js
+const myData = [
```

Comment:
fix the alignment of these keys (ie. the name of each key should be vertically aligned) and don't use `const`